### PR TITLE
Fix crafting overlay

### PR DESCRIPTION
**Pull request by romangraef**

### CODE REVIEW

**Review by NopoTheGamer — DISMISSED** [review]
works

**Review by Lulonaut — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 1]
```diff
@@ -127,16 +131,17 @@ public JsonObject getJsonFromFile(File file) {
     public void resetRepo() {
         try {
             Utils.recursiveDelete(new File(configLocation, "repo"));
-        } catch (Exception ignored) {}
+        } catch (Exception ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:134]: aaaaaaaaaaaaaa

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 2]
```diff
@@ -398,7 +398,8 @@ public static Method getMethod(Class<?> clazz, Class<?>[] params, String... meth
         for (String methodName : methodNames) {
             try {
                 return clazz.getDeclaredMethod(methodName, params);
-            } catch (Exception ignored) {}
+            } catch (Exception ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:401]: more aaaaaaaaa

**Review by IRONM00N — CHANGES_REQUESTED** [review]
please fix the empty catch closing brace being on a new line

**Comment by jani270** [comment]
PepeLaugh

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 3]
```diff
@@ -938,7 +945,8 @@ public void writeItemToFile(ItemStack stack) {
 
         try {
             writeJson(json, new File(new File(repoLocation, "items"), internalname + ".json"));
-        } catch (IOException ignored) {}
+        } catch (IOException ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:948]:
```suggestion
        } catch (IOException ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 4]
```diff
@@ -938,7 +945,8 @@ public void writeItemToFile(ItemStack stack) {
...
+        }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:949]:
```suggestion
```

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 5]
```diff
@@ -1400,7 +1406,8 @@ public ItemStack jsonToStack(JsonObject json, boolean useCache, boolean useRepla
                 try {
                     NBTTagCompound tag = JsonToNBT.getTagFromJson(json.get("nbttag").getAsString());
                     stack.setTagCompound(tag);
-                } catch (NBTException ignored) {}
+                } catch (NBTException ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:1409]:
```suggestion
                } catch (NBTException ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 6]
```diff
@@ -1400,7 +1406,8 @@ public ItemStack jsonToStack(JsonObject json, boolean useCache, boolean useRepla
...
+                }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:1410]:
```suggestion
```

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 7]
```diff
@@ -409,13 +409,15 @@ public static Object getField(Class<?> clazz, Object o, String... fieldNames) {
             try {
                 field = clazz.getDeclaredField(fieldName);
                 break;
-            } catch (Exception ignored) {}
+            } catch (Exception ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:412]:
```suggestion
            } catch (Exception ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 8]
```diff
@@ -409,13 +409,15 @@ public static Object getField(Class<?> clazz, Object o, String... fieldNames) {
...
+            }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:413]:
```suggestion
```

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 9]
```diff
@@ -409,13 +409,15 @@ public static Object getField(Class<?> clazz, Object o, String... fieldNames) {
...
         }
         if (field != null) {
             field.setAccessible(true);
             try {
                 return field.get(o);
-            } catch (IllegalAccessException ignored) {}
+            } catch (IllegalAccessException ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:419]:
```suggestion
            } catch (IllegalAccessException ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 10]
```diff
@@ -409,13 +409,15 @@ public static Object getField(Class<?> clazz, Object o, String... fieldNames) {
...
+            }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:420]:
```suggestion
```

**Review by IRONM00N — CHANGES_REQUESTED** [review]
:trollface:

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 11]
```diff
@@ -255,7 +259,8 @@ public void loadItemInformation() {
                         newCurrentCommitJSON.addProperty("sha", latestRepoCommit);
                         try {
                             writeJson(newCurrentCommitJSON, new File(configLocation, "currentCommit.json"));
-                        } catch (IOException ignored) {}
+                        } catch (IOException ignored) {
+                        }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:263]:
```suggestion
                        } catch (IOException ignored) {}
```

**Review by IRONM00N — CHANGES_REQUESTED** [review]

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**